packages/wasm-dpp/test/integration/identity/IdentityFacade.spec.js

```
@@ -62,16 +62,11 @@
 
   describe('#createFromBuffer', () => {
     it('should create Identity from a Buffer', () => {
-      let result;
-      try {
-        result = dpp.identity.createFromBuffer(identity.toBuffer());
-      } catch (e) {
-        console.dir(e.getErrors()[0].toString());
-      }
+      const identity = dpp.identity.createFromBuffer(identity.toBuffer());
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)